R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
When specifying schema with `bigint`, we will hit the bigint path. Without this change, it will thrown an error of type mismatch. But as you said, we can't specify `bigint` type in R console.